requirements.txt

```diff
@@ -0,0 +1,2 @@
+# file showing needed dependancies for runs
+server-groups.yaml needs 'ansible-galaxy collection install openstack.cloud' ansible 2.9 or greater
```

Comment:
This file could be a yaml file with this in.

```yaml
---
collections:
- name: openstack.cloud
  version: "1.2.0"
```

See https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#installing-roles-and-collections-from-the-same-requirements-yml-file

Reply:
Nice, this has been added along with a requirements install line to the deploy.sh